client/lcd/lcd_test.go

```diff
@@ -196,7 +196,7 @@ func TestCoinSend(t *testing.T) {
 	require.Equal(t, http.StatusInternalServerError, res.StatusCode, body)
 
 	// test failure with wrong adjustment
-	res, body, _ = doTransferWithGas(t, port, seed, name1, pw, addr, "simulate", 0.1, false, false)
+	res, body, _ = doTransferWithGas(t, port, seed, name1, pw, addr, "auto", 0.1, false, false)
```

Comment:
Should we use `GasFlagAuto` constant here?

Reply:
ditto